app/views/request-to-go-live/index.njk

```diff
@@ -6,29 +6,32 @@
 
 {% block mainContent %}
   {% if showNextSteps %}
-    <div class="govuk-grid-column-full">
-    {{ govukPanel({
-      titleText: "Request submitted"
-    }) }}
-    </div>
     <div class="govuk-grid-column-two-thirds">
-      <h3 class="govuk-heading-m govuk-!-margin-top-4">What happens next</h3>
-      <p class="govuk-body">We’ll email you in one working day with a link to your live account.</p>
-      <p class="govuk-body">Sign in and follow the account set up instructions.</p>
-      <p class="govuk-body">You’ll need to provide:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        {% if pspIsStripe %}
-          <li>your organisation’s contact details</li>
+      <h1 class="govuk-heading-l">Request submitted</h1>
+
+      <p class="govuk-body govuk-!-font-size-24">We’ll respond within one working day, and we can usually activate your live account on the same day.</p>
+
+      {% if pspIsStripe %}
+        <h2 class="govuk-heading-m govuk-!-margin-top-4">What happens next</h2>
+
+        <p class="govuk-body">When we approve your request your service can start taking payments from users immediately.</p>
+
+        <p class="govuk-body">To get money paid into your bank account from Stripe, you need to add more details. You need to provide:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>the name, date of birth and home address of the person in your organisation legally reponsible for payments (called your ‘responsible person’)</li>
           <li>your bank details</li>
           <li>your VAT number</li>
-          <li>your company number (if you have one)</li>
-          <li>the name, date of birth and home address of the person in your organisation legally responsible for payments (called your ‘responsible person’)</li>
-        {% else %}
-          <li>your organisation’s contact details</li>
-          <li>account credentials related to your payment service provider</li>
-        {% endif %}
-      </ul>
-      <p class="govuk-body">Once that’s done, you can start taking payments.</p>
+        </ul>
+
+        <p class="govuk-body">GOV.UK Pay will email you a link to add these additional details after approving your request.</p>        
+
+        <p class="govuk-body">All users with admin permissions can do this.  You can <a href="/service/{{ currentService.externalId }}" class="govuk-link">manage your users</a> here.</p>
+      {% else %}}
+        <p class="govuk-body">When we activate your live account, we’ll email you a link with next steps.</p>
```

Comment:
Are we emailing them a link for BYOC?

Reply:
Good spot - updated to say this now:
![image](https://user-images.githubusercontent.com/59831992/85563351-348e2f00-b625-11ea-83ab-ef50f957dd54.png)